11/umbraco-cms/extending/filesystemproviders/azure-blob-storage.md

```diff
@@ -96,9 +98,8 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         .WithMiddleware(u =>
```

Comment:
Also, is this hint about Umbraco 9 still needed for the v11 version of the docs?

Should it be replaced with a hint explaining the difference between the v10.0.0 and v11.0.0 versions of `Umbraco.StorageProviders.AzureBlob`? i.e. the fact that ImageSharp cache support has been split out.
As someone upgrading their site, from Umbraco 10 to 11, might be confused as to why their ImageSharp cache suddenly stops uploading to their blob storage.

Reply:
Yes I think we can remove the hint about being in v9, (as this is v10 documentation now), and instead replace it with what you're suggesting 💪 Are you up for that ? 🤞

Reply:
@Zeegaan I've replaced the hint, let me know if it's any good 😄